__tests__/app.test.js

```diff
@@ -467,3 +467,41 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles (sorting)", () => {
+  test("200: sorts articles by specific column in ascending order", () => {
+    return request(app)
+      .get("/api/articles?sort_by=title&order=asc")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toBeSortedBy("title", { ascending: true });
```

Comment:
just a heads up that jest sorted doesnt have ascending true but it defaults to descending false which is why this is working - I would pop it in with the descending false instead :)